blog/_posts/2022-02-24-rancher-desktop-container-runtimes.md

```diff
@@ -0,0 +1,118 @@
+---
+slug: "rancher-desktop-container-runtimes"
+date: 2022-02-24
+author: milas
+layout: blog
+title: "Rancher Desktop: Should You Use containerd Or dockerd?"
+image: "/assets/images/rancher-desktop-container-runtimes/title.jpg"
+image_caption: 'Photo by <a rel="noopener noreferrer" href="https://unsplash.com/photos/u0vgcIOQG08">Jens Lelie</a> on Unsplash'
+description: "Configure Tilt to build images with kim for use with Rancher Desktop"
+tags:
+- docker
+- moby
+- containerd
+- rancher
+- kubernetes
+---
+
+[Rancher Desktop][rancher-desktop], a lightweight and local Kubernetes cluster solution, has been evolving _fast_ over the past few months!
+
+Early releases used [containerd][] behind-the-scenes with an experimental CLI tool, [`kim`][kim], to build images.
+I wrote up a blog post about how easy it was to integrate with Tilt: [Writing Yet Another Custom Image Builder][blog-kim-extension].
+
+Since then, however, Rancher Desktop has started also supporting Docker (aka `dockerd (moby)`) as an alternative to containerd.
```

Comment:
I would probably get rid of mention of `moby`. I've never heard a user call it that. I'm not sure I've even heard the Docker people use it since the company split up.